_commands/basics/rm.md

```diff
@@ -0,0 +1,55 @@
+---
+---
+
+rm
+--
+
+`rm` is used to delete one or multiple files from your current directory. 
+
+~~~ bash
+$ rm file1.txt file2.txt
+
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `rm -d`
```

Comment:
I always do `rm -R`. I found [this](http://stackoverflow.com/questions/11976861/bash-whats-the-difference-between-rm-d-and-rm-r). Also in a `man rm` got 

           -d, --dir
              remove empty directories

which contradicts your explanation. Maybe @ppannuto can clear up some of the ambiguity?